SYCL/ESIMD/spec_const/spec_const_ulong.cpp

```diff
@@ -0,0 +1,21 @@
+//==--------------- spec_const_ulong.cpp  - DPC++ ESIMD on-device test ----===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu
+// XFAIL: windows
+// RUN: %clangxx-esimd -fsycl -I%S/.. %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// UNSUPPORTED: cuda
+
+#define DEF_VAL 0xdeaddeaf4badbeaf
+#define REDEF_VAL 0x4cafebad00112233
+#define STORE 1
+
+typedef unsigned long spec_const_t;
```

Comment:
unsigned long is 32 bit on Windows. 
```suggestion
using spec_const_t = uint64_t;
```